packages/launch/preset/_assets/markdown.css

```diff
@@ -137,7 +137,6 @@
 
 .markdown-body a {
   color: var(--markdown-link-color, #0366d6);
-  text-decoration: none;
```

Comment:
no text-decoration is how markdown get's rendered on GitHub... It is probably better to stay aligned - if you get GitHub to change then we should follow

Reply:
With text-decoration is how links get rendered by HTML? 

I don't understand how GitHub is the guideline for your decisions. What GitHub choses, they must decide for themselves. And whatever GitHub does, I still think it would improve readability. 
I think underlining is an improvement for the usability of Rocket.

> Textual links should be colored and underlined to achieve the best perceived affordance of clickability, though there are a few exceptions to these guidelines.
https://www.nngroup.com/articles/guidelines-for-visualizing-links/

From https://adrianroselli.com/2016/06/on-link-underlines.html

Reply:
Any update on this? We're gonna need to fix it, on one level or another. 😕

https://www.w3.org/WAI/WCAG21/Techniques/general/G183

Reply:
Hi @daKmoR, do you still want to be aligned with GitHub? 😄

https://github.blog/changelog/2023-10-18-new-default-underlined-links-for-improved-accessibility/
(this PR was so ahead of its time)